localstack/services/apigateway/integration.py

```diff
@@ -466,7 +483,6 @@ def invoke(self, invocation_context: ApiInvocationContext):
             # integration type "AWS" is only supported for WebSocket APIs and REST
             # API (v1), but the template selection expression is only supported for
             # Websockets
-            template_key = None
             if invocation_context.is_websocket_request():
                 template_key = invocation_context.integration.get(
                     "TemplateSelectionExpression", "$default"
```

Comment:
q: should this also be replace with `render_template_selection_expression`?

Reply:
yes, can be done on this PR as well.